front_end/css/styles.css

```diff
@@ -79,7 +79,7 @@ pre {
 
```

Comment:
These changes address long course titles spilling over on the landing page. Because allowing their width to automatically scale changes the number of courses that fit onto each line, I had to center the box of courses. Let me know if this is a change you'd like to keep, or whether you like it better the way it used to be. 
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/61362908/129926550-6a188c72-fb53-456d-a516-16937f063f42.png">

Reply:
Yes, I think it's a fine compromise to do it this way. Will you please just leave a comment in the code that indicates what we would change if we wanted to go back to the old way?